consensus/src/main/java/org/ethereum/beacon/consensus/verifier/operation/AttestationVerifier.java

```diff
@@ -27,8 +28,9 @@ public VerificationResult verify(Attestation attestation, BeaconState state) {
     try {
       spec.verify_attestation(state, attestation);
       return VerificationResult.PASSED;
-    } catch (Exception e) {
-      return VerificationResult.failedResult(e.getMessage());
+    } catch (SpecCommons.SpecAssertionFailed e) {
+      String error = e.getStackTrace().length > 0 ? e.getStackTrace()[1].toString() : "SpecAssertion";
```

Comment:
Literally, `e.getStackTrace().length > 0` check doesn't guarantee that `e.getStackTrace()[1]` won't turn with index out of bounds; but I believe it's OK for this particular case.

Do you mind overriding `getMessage` for `SpecAssertionFailed` which would execute this line of code and as a result return `error` string?